### PR TITLE
Updates to monitoring/alerts standard based on #61

### PR DESCRIPTION
**Pull request by kfriedman**
Hi!

Where was some feedback in #61 about updating the Monitoring & Alarms standard based on the incident.

This PR attempts to address this feedback by:
- Reorganizing the standard a bit so it has a more logical flow
- Adding a small "Testing" section based on the feedback from @thisisstephenbetts  and @nonword. Hopefully we can iterate on this.

Thanks!

### CODE REVIEW

File: standards/alerting.md [hunk 1]
```diff
@@ -27,7 +46,8 @@ The metric created from the metric filter MUST be used to trigger an alarm.
 
 2. Go to `CloudWatch` by clicking `Services` dropdown menu from the top navigation and choose `CloudWatch`.
 
-3. Go to `Logs` from the left navigation and find the log you want to build the meric on from the `Log Groups` list.
+3. Go to `Logs` from the left navigation and find the log group you want to build the metric on from the `Log Groups` list.
+   - Warning: Be careful about choosing the correct log group - especially for Elastic Beanstalk log groups (see a [previous post-mortem](../post-morterms/2018-03-30-sierra-api.md) that had metric filters improperly configured).
```
Comment (nodanaonlyzuul) [inline standards/alerting.md]: Can we:

1. Change that folder name to "post-mortems", not "post-morterms".
1. Change this link back to 'post-mortems"

That "r" shouldn't be there.

Comment (nonword) [reply standards/alerting.md]: i wish the plural was posts mortem

Comment (thisisstephenbetts) [reply standards/alerting.md]: :-D

Comment (kfriedman) [reply standards/alerting.md]: I don't know why but I kept wanting to write "post-porte*r*ms"!

Anyway, thanks for pointing that out. I fixed the typos.

**Pull request merged**